tools/genesis-snapshot/snapshotcreator/snapshotcreator.go

```diff
@@ -56,7 +56,6 @@ func CreateSnapshot(genesisTokenAmount uint64, genesisSeedBytes []byte, nodesToP
 
 	return &snapshot.Snapshot{
 		LedgerSnapshot: ledger.NewSnapshot(outputs, outputsMetadata),
```

Comment:
`manaSnapshot := mana.NewSnapshot()` above can also be removed as it is no longer used